tests/test_examples.py

```diff
@@ -1,14 +1,11 @@
-import os
-
 import github
 import pytest  # type: ignore
 import requests  # type: ignore
 from pydantic import ValidationError
 
 from ethpm_types import PackageManifest
 
-# NOTE: Must export GITHUB_ACCESS_TOKEN
-ETHPM_SPEC_REPO = github.Github(os.environ["GITHUB_ACCESS_TOKEN"]).get_repo("ethpm/ethpm-spec")
```

Comment:
Ahhhh, okay, so if this token is present, you should use it because of the github rate limit without the token